public/app/plugins/panel/barchart/bars.ts

```diff
@@ -226,7 +226,7 @@ export function getConfig(opts: BarsOptions, theme: GrafanaTheme2) {
 
       // Calculate final labels positions according to unified text size
       const textMeasurement = measureText(label.formattedValue, fontSize * devicePixelRatio);
-      const actualLineHeight = textMeasurement.fontBoundingBoxAscent + textMeasurement.fontBoundingBoxDescent;
+      const actualLineHeight = textMeasurement.actualBoundingBoxAscent + textMeasurement.actualBoundingBoxDescent;
```

Comment:
nit: Is this change by accident or on purpose? Thinking that we are changing the properties being used?

Reply:
We can test impact of this change(if it's not accidental) in this dashboard: http://localhost:3000/d/WFlOM-jM1/barchart-panel-tests-value-sizing?orgId=1

Reply:
This change seems to bring regression to bar chart value size alignment: 

| Before | After |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/2376619/122215076-1f03bc00-ceab-11eb-8d3b-1ef1499bcd5c.png) | ![image](https://user-images.githubusercontent.com/2376619/122214273-31c9c100-ceaa-11eb-9c93-8d84c2efc4d0.png) |

Reply:
So the reason for this change is that 4.3.2 no longer supports this property. I couldn't find any details in typescript changelogs or blog posts as to why this was done, but I suspect it has to do with the fact that it's a non-standard property and isn't supported outside of Chrome (https://caniuse.com/?search=fontBoundingBoxAscent). I'll look into workarounds though.